ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```diff
@@ -150,6 +150,14 @@ images:
   dockerfile_path: Dockerfile
   from: intel-runtime-ml-ubi9-python-3.9
   to: jupyter-intel-ml-ubi9-python-3.9
+- context_dir: amd/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: base-ubi9-python-3.9
+  to: amd-ubi9-python-3.9
+- context_dir: jupyter/minimal/ubi9-python-3.9
+  dockerfile_path: Dockerfile
+  from: amd-ubi9-python-3.9
+  to: amd-jupyter-minimal-ubi9-python-3.9
```

Comment:
As we are not building UBI based image  in the first interaction, we can remove these.